doc/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -84,7 +97,7 @@ We will have to generate a YAML configuration file for configuring the 3D sensor
 
 
 Update the launch file
-----------------------
+++++++++++++++++++++++
 
 Add the YAML file to the launch script
```

Comment:
Rename the `sensors_kinect.yaml` to point to the correct `.yaml`

Reply:
Also, I think that `sensor_manager.launch` is incorrect. I think it should be `<robot_name>_moveit_sensor_manager.launch.xml` or but you should double check.

Reply:
at least in panda it is sensor_manager.launch